Pj_MonteVerdi_defender.gmx/objects/obj_pattern_stairs_parent.object.gmx

```diff
@@ -0,0 +1,49 @@
+<!--This Document is generated by GameMaker, if you edit it by hand then you do so at your own risk!-->
+<object>
+  <spriteName>&lt;undefined&gt;</spriteName>
+  <solid>0</solid>
+  <visible>-1</visible>
+  <depth>0</depth>
+  <persistent>0</persistent>
+  <parentName>&lt;undefined&gt;</parentName>
+  <maskName>&lt;undefined&gt;</maskName>
+  <events>
+    <event eventtype="0" enumb="0">
+      <action>
+        <libid>1</libid>
+        <id>603</id>
+        <kind>7</kind>
+        <userelative>0</userelative>
+        <isquestion>0</isquestion>
+        <useapplyto>-1</useapplyto>
+        <exetype>2</exetype>
+        <functionname></functionname>
+        <codestring></codestring>
+        <whoName>self</whoName>
+        <relative>0</relative>
+        <isnot>0</isnot>
+        <arguments>
+          <argument>
+            <kind>1</kind>
+            <string>hspc = 5;
+vspc = 6;
+bspd = 20;
```

Comment:
try to give the variables more meaningful names. Now is hard to get what those variables mean